streamer/src/quic.rs

```diff
@@ -514,7 +516,14 @@ mod test {
             .with_safe_defaults()
             .with_custom_certificate_verifier(SkipServerVerification::new())
             .with_no_client_auth();
-        ClientConfig::new(Arc::new(crypto))
+        let mut config = ClientConfig::new(Arc::new(crypto));
+
+        let transport_config = Arc::get_mut(&mut config.transport).unwrap();
+        let timeout = IdleTimeout::from(VarInt::from_u32(60_000));
```

Comment:
While we currently have a per-ip connection limit, if we were to start sending keepalive packets and have a longer timeout, we would likely need a global connection limit for the server as well to prevent denial of service attacks.

Reply:
Never mind, seems like there's already a cap.